enterprise/api_client/ecommerce.py

```diff
@@ -27,7 +29,7 @@ class EcommerceApiClient(object):
     Object builds an API client to make calls to the E-Commerce API.
     """
 
-    def __init__(self, user):
+    def __init__(self, user=None):
```

Comment:
i'd rather have a static method that generates a client with the ecommerce service user than have that be the default behavior of initialization